riak/tests/comparison.py

```diff
@@ -1,20 +1,4 @@
-"""
-Copyright 2014 Basho Technologies, Inc.
-
-This file is provided to you under the Apache License,
-Version 2.0 (the "License"); you may not use this file
-except in compliance with the License.  You may obtain
-a copy of the License at
-
-  http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing,
-software distributed under the License is distributed on an
-"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-KIND, either express or implied.  See the License for the
-specific language governing permissions and limitations
-under the License.
-"""
+# -*- coding: utf-8 -*-
```

Comment:
Yeah, they're not necessary. I have been removing them as I go from the .NET and Node.js source files too. The top level LICENSE is sufficient.